program-runtime/src/prioritization_fee.rs

```diff
@@ -5,8 +5,6 @@ type MicroLamports = u128;
 
 pub enum PrioritizationFeeType {
```

Comment:
Do we plan to introduce more variants of prioritization in the near future?
If not, imo we should just get rid of this enum entirely.

Reply:
maybe just an issue?

Reply:
#34301